substrate/frame/identity/src/lib.rs

```diff
@@ -192,6 +211,11 @@ pub mod pallet {
 		#[pallet::constant]
 		type PendingUsernameExpiration: Get<BlockNumberFor<Self>>;
 
+		/// The number of blocks that must pass to enable the permanent deletion of a username by
+		/// its respective authority.
```

Comment:
Can this be dynamically changed?  
(I am thinking about the parameters pallet)

Reply:
I guess it could? I don't know if there's any value in having this be dynamic, we'd ideally want this to be as stable as possible so it's predictable for wallets and users alike (they'd put this in their ToS).